### PR TITLE
Dissociating file and output format

### PR DESCRIPTION
**Pull request by D-XIII**
Added an option to dissociate the format of the output in stdout and the output file.

Option added : 
-file-format or -ff

### CODE REVIEW

**Comment by Hackndo** [comment]
Hey there, great PR! I'd like to merge it in branch 3.1.2, but there are 2 conflicts. Could you solve them before I can merge? 

Thank you!

File: lsassy/writer.py [hunk 1]
```diff
@@ -28,16 +31,27 @@ def get_output(self, out_format, users_only=False):
...
         :param out_format: Output format
         :param output_file: Output file
+        :param file_format: File Logs Format
         :param quiet: If set, doesn't display on stdout
         :param users_only: If set, only returns users account, else returns users and computers accounts
         :return: Success status
         """
         output = self.get_output(out_format, users_only)
+        
+        if file_format is null:
```
Comment (Hackndo) [inline lsassy/writer.py:46]: Why do you use `from sqlalchemy import null` for this, and not `None`?

**Review by Hackndo — CHANGES_REQUESTED** [review]
I tested with "None" instead of the imported "null" and it seems to work great. When you validate and update, and update the conflicts, I'll merge, and merge this branch into master 👍

File: lsassy/writer.py (hunk 1 above)
Comment (D-XIII) [reply lsassy/writer.py:46]: That's weird that I putted null instead of None.
Seems to be auto imported by my IDE. I'm gonna change that.

**Pull request merged**